hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/TestRMHA.java

```diff
@@ -610,7 +613,7 @@ protected void serviceStart() throws Exception {
   }
 
   @Test
-  @Timeout(value = 9000)
+  @Timeout(value = 9000) // FIXME that's more than two hours
```

Comment:
I thought maybe this was just 9000 millis with a new default, but no, the old timeout was 900000 mills, so now its just in a different unit.

I think the value here is to ensure that it never hangs the build forever, its just a very big one. Looking at the history of that line, YARN-4927 multiplied the timeout up from 90 seconds, so it was probably just "a number which made the failures go away".

I'd propose 10 minutes.